apps/protocol-frontend/src/components/BulkAttestationModal.tsx

```diff
@@ -13,26 +13,36 @@ import { useUser } from '../contexts/UserContext';
 import { MdCheckCircle } from 'react-icons/all';
 import { AttestationTableType } from '../types/table';
```

Comment:
(random spot to start thread), when bulk attesting shouldn't the attest button become bulk attest?

Reply:
Modal doesn't close when attesting is done